### PR TITLE
Second

### PR DESCRIPTION
**Pull request by pavitrachavda97**
## Description 📝

Closes: #392 
Added Scroll to top button

## Changes 💬
- Created ScrollToTop in components
- Created ScrollToTopPage inside ScrollToTopfolder
- Added ScrollToTop component in index.tsx

## Screenshots (if applicable)
Light Mode:
image

Dark Mode:
image

### CODE REVIEW

File: components/ScrollToTop/ScrollToTopPage.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  width: '50px',
+  height: '50px',
+  backgroundColor: '#333',
+  color: '#fff',
+  textAlign: 'center',
+  lineHeight: '50px',
+  fontSize: '20px',
+  cursor: 'pointer',
+  zIndex: 9999,
+};
```
Comment (PiyushKalyanpy) [inline components/ScrollToTop/ScrollToTopPage.tsx:17]: Please use tailwind css instead of this.

File: package-lock.json
Comment (PiyushKalyanpy) [inline package-lock.json:1]: Remove this file. 
#466

File: components/ScrollToTop/ScrollToTopPage.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      behavior: 'smooth',
+    });
+  };
+
+  return (
+    <div
+      style={{ ...scrollToTopButtonStyle, display: showButton ? 'block' : 'none' }}
+      onClick={scrollToTop}
+    >
+      &uarr;
```
Comment (PiyushKalyanpy) [inline components/ScrollToTop/ScrollToTopPage.tsx:50]: User google material icons here. 
`<span className="material-symbols-outlined>{icon name}</span>`

File: components/ScrollToTop/ScrollToTopPage.tsx
Comment (PiyushKalyanpy) [inline components/ScrollToTop/ScrollToTopPage.tsx:1]: Move this file from ScrollToTop folder to Utils folder inside compoenents.

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

**Comment by pavitrachavda97** [comment]
@PiyushKalyanpy 
I have made the requested changes. The changes have been committed and pushed to the repository. Please review the updated code and let me know if any further adjustments are needed. Thank you!

**Pull request closed without merging**